jenkins/image_building/dib_elements/ubuntu-ci/install.d/55-install

```diff
@@ -34,11 +34,25 @@ sudo echo \
   $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list
 
 sudo apt-get update
-sudo apt-get install docker-ce docker-ce-cli containerd.io jq -y
+sudo apt-get install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin jq -y
 sudo groupadd docker || true
 sudo usermod -aG docker metal3ci || true
 sudo systemctl enable docker
+
+# Configure Docker daemon with BuildKit support
+sudo mkdir -p /etc/docker
+sudo tee /etc/docker/daemon.json <<EOF
```

Comment:
We have [ansible templates injecting daemon.json](https://github.com/metal3-io/metal3-dev-env/blob/a71df8326aa26963beac88af7950759013bebcec/vm-setup/roles/packages_installation/tasks/ubuntu_required_packages.yml#L133) as well in dev-env. 
https://github.com/metal3-io/metal3-dev-env/blob/a71df8326aa26963beac88af7950759013bebcec/vm-setup/roles/packages_installation/files/daemon.json#L1

Reply:
OTOH, we want as much stuff installed here so the CI runs are faster. The more we leave to dev-env the slower (and flakier) the tests are.

Edit: misread your comment, you said same basically 🙂

Reply:
I got a bit lost in the layers. Is it so that the container image build that failed is coming from metal3-dev-env? I.e. we test the ironic-image through metal3-dev-env and that is where it is being built?
I am fine solving this through metal3-dev-env, but I hate how invasive metal3-dev-env is...
It would be nice to be able to have it in the CI image here and still work together with metal3-dev-env.